### PR TITLE
Nag for deprecation of remaining members of VersionNumber

### PR DESCRIPTION
**Pull request by eskatos**
* Fixes https://github.com/gradle/gradle/issues/21902

### CODE REVIEW

**Comment by eskatos** [comment]
@bot-gradle test AST

**Comment by eskatos** [comment]
@bot-gradle test this

File: subprojects/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc [hunk 1]
```diff
@@ -47,6 +47,17 @@ The `groovy-base` plugin is now responsible for configuring source and target co
 If you are using this task *without applying `grooy-base`*, you will have to manually set compatibility versions on these tasks.
 In general, the `groovy-base` plugin should be applied whenever working with Groovy language tasks.
 
+=== Deprecations
+
+[[org_gradle_util_reports_deprecations]]
+==== Deprecated members of the `org.gradle.util` package now report their deprecation
+
+These members will be removed in Gradle 9.0.
```
Comment (lkasso) [inline subprojects/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc:55]:
```suggestion
These members will be removed in Gradle 9.0:
```

**Review by lkasso — APPROVED** [review]
Docs approved.

**Review by ov7a — APPROVED** [review]

**Comment by eskatos** [comment]
@bot-gradle test and merge

**Comment by eskatos** [comment]
@bot-gradle test and merge

**Pull request merged**